src/app/_utils/code-example.ts

```diff
@@ -20,7 +20,7 @@ import {CodeHighlight} from "../../clarity-angular/code/syntax-highlight/syntax-
     `]
 })
 export class CodeExample {
-    @ViewChild(CodeHighlight) codeHighlight: CodeHighlight;
+    @ViewChild(ClrCodeHighlight) codeHighlight: ClrCodeHighlight;
```

Comment:
Unrelated to your change: I don't even remember why we have to query code highlights all over the place. But not only do we not need it for the demo app, this directive was also written in a few minutes and was never meant to make it in prod, let alone in our public API.

Had it been fixed before, we wouldn't have needed to rename it everywhere. 😢

Reply:
If it was never intended, we can deprecate it?

Reply:
Well, apparently people now completely depend on it, including the Prism dependency and all. We've had requests for bugfixes on this component.

So rather than deprecate it, I think we should own our mistake and clean it up to make it an up-to-standards Clarity component. But as usual, the priority on that is super low.